playbooks/tests/tasks/ceilometer.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: test ceilometer
+  hosts: controller[0]
+  tasks:
+  - name: list ceilometer meters
+    shell: . /root/stackrc; ceilometer meter-list
+    register: private_subnet
```

Comment:
Why is this being registered? Does not look like the registered value is being used anywhere